src/lib/fetchChatter.ts

```diff
@@ -30,9 +30,9 @@ export class FetchChatter {
             'chatter');
     }
 
-    list(): Promise<any> {
```

Comment:
@kidtsunami happy for you to keep legacy methods, but "list" doesn't make much sense to the average salesforce developer. the [chatter REST API is extensive](https://developer.salesforce.com/docs/atlas.en-us.chatterapi.meta/chatterapi/connect_resources_overview.htm), ([common calls](https://developer.salesforce.com/docs/atlas.en-us.chatterapi.meta/chatterapi/quickreference.htm)) I'd suggest either renaming legacy methods to be more specific, e.g. `getMyNewsFeed` or scrapping helper methods altogether in favor of letting the developer build a wrapper to this library.